src/test/run-make/json-errors/Makefile

```diff
@@ -0,0 +1,8 @@
+-include ../tools.mk
+
+all:
+	cp foo.rs $(TMPDIR)
+	cd $(TMPDIR)
+	$(RUSTC) -Z unstable-options --error-format=json foo.rs 2>foo.log || true
```

Comment:
You can ignore return status by prefixing line with a `-`, like this `-$(RUSTC) ...`